components/modal/layer-info-modal/layer-info-modal-component.js

```diff
@@ -1,55 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'routes';
 
 // Redux
 import { connect } from 'react-redux';
 
-class LayerInfoModal extends React.Component {
-  static propTypes = {
-    layer: PropTypes.object,
-    embed: PropTypes.bool
-  };
-
-  render() {
-    const { embed, layer } = this.props;
-
-    return (
-      <div className="layer-info-modal">
-        <div className="layer-info-content">
-          <h2>{layer.name}</h2>
-          <p>{layer.description}</p>
-          <div className="buttons">
-            {embed &&
-              <a
-                className="c-btn -primary"
-                href={`${window.location.origin}/data/explore/${layer.dataset}`}
-                target="_blank"
-              >
-                More info
-              </a>
-            }
-
-            {!embed &&
-              <Link
-                route="explore_detail"
-                params={{ id: layer.dataset }}
-              >
-                <a className="c-btn -primary">
-                  More info
-                </a>
-              </Link>
-            }
-          </div>
+import { fetchDataset } from 'services/dataset';
+
+function LayerInfoModal(props) {
+  const { embed, layer } = props;
+  const [slug, setSlug] = useState(' ');
```

Comment:
`(' ')` what's this?

Reply:
If we use the empty string instead, there's the following error thrown when the `<Link>` is built as part of the render:
![image](https://user-images.githubusercontent.com/545342/61215263-a1f1ca00-a70a-11e9-9ae8-5ca1bd8c32e9.png)